src/Candidate.js

```diff
@@ -0,0 +1,36 @@
+import React, {useState, useEffect} from 'react';
+import styled from 'styled-components';
+
+function Candidate({ name, voteCount, cid, rank, vote }) {
+  function clickTest() {
+    vote(cid, name);
+  }
+
+  return (
+    <Wrapper>
+      <span>{rank+1}위:</span>
+      <CandidateAndVote>
```

Comment:
`CandidateAndVote`보다는 이런 네이밍이 좀 더 적절할 것 같아요 :)

```suggestion
      <CandidateCard>
```

Reply:
적절한 이름이 생각안나서 큰일이었어요..! Card도 좋은 단어네요 !